sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/AnsiUtil.scala

```diff
@@ -44,12 +43,15 @@ object AnsiUtil extends Arm {
   }
 
   private def castDoubleToTimestampAnsi(doubleInput: ColumnView, toType: DataType): ColumnVector = {
-    val msg = s"The column contains at least a single value that is " +
-        s"NaN, Infinity or out-of-range values. To return NULL instead, use 'try_cast'. " +
-        s"If necessary set ${SQLConf.ANSI_ENABLED.key} to false to bypass this error."
+    val msg = s"The column contains out-of-range values. To return NULL instead, use " +
+        s"'try_cast'. If necessary set ${SQLConf.ANSI_ENABLED.key} to false to bypass this error."
```

Comment:
nit: in the comment it looks like the original Spark message uses quotes around the config key https://github.com/NVIDIA/spark-rapids/pull/5731#issuecomment-1146218794

Reply:
Created follow on issue : https://github.com/NVIDIA/spark-rapids/issues/5750 for changing all the error messages based on above format.